### ISSUE TITLE
[EDITOR] Unsaved Changes on Exit

### ISSUE THREAD

**Issue by G-Ambatte** [naturalcrit/homebrewery#3238]
### Your idea:

Reproduction Steps:

1. Open any document.
2. Make a change. Note that the "AUTOSAVE" icon changes to "SAVE NOW"
3. Revert that change. Note that the "SAVE NOW" reverts to "AUTOSAVE"
4. Attempt to close the tab.

Despite the fact that there are no changes to be saved, the "Your changes may be lost" warning appears.

![image](https://github.com/naturalcrit/homebrewery/assets/19826659/6afcc429-7767-4b16-9f0a-16cf9745ecd8)

Solution: update the function called at editor close to use the same logic as the SAVE button.

**Comment by dbolack-ab** [comment]
Isn't this mitigated now?

**Comment by 5e-Cleric** [comment]
Yes this is fixed, not sure when, probably calc's rework of this system
